integration/cmd_serve_test.go

```diff
@@ -11,8 +11,6 @@ import (
 )
 
 func TestServeLaunchpadAppWithWasm(t *testing.T) {
-	t.Parallel()
```

Comment:
Pure curiosity: why?

Reply:
It was related to reduce the cpu usage on CI. Because parallel tests create a huge stress on it and our tests tends to timeout.